tests/test_validators.py

```diff
@@ -521,7 +521,7 @@ class UniqueConstraintModel(models.Model):
     race_name = models.CharField(max_length=100)
     position = models.IntegerField()
     global_id = models.IntegerField()
-    fancy_conditions = models.IntegerField(null=True)
```

Comment:
why the null=True is removed? can you explain a bit more, please?

Reply:
With recent changes null=True generate default value for that field as a result I can't test that this field is required if it present in condition. I don't think null=True has any meaning here so I just removed it instead of creating a new test model.